### PR TITLE
Fix small typos

### PR DESCRIPTION
**Pull request by tilsammans**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Include javascripts and stylesheets:
     //= require "pageflow/internal_links"
 
     # app/assets/javascripts/pageflow/editor.js
-    //= require pageflow/internal_links/editor
+    //= require "pageflow/internal_links/editor"
```
Comment (tf) [inline README.md]: Sprockets does not use quotes in requires, does it?

Comment (tilsammans) [inline README.md]: Ah, I guess not.

In that case the edit should change the line above it. I prefer consistency. :)

Op 19 aug. 2015, om 13:34 heeft Tim Fischbach <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:

In README.mdhttps://github.com/codevise/pageflow-internal-links/pull/2#discussion_r37404568:

> @@ -22,7 +22,7 @@ Include javascripts and stylesheets:
>      //= require "pageflow/internal_links"
> 
> ```
>  # app/assets/javascripts/pageflow/editor.js
> ```
> -    //= require pageflow/internal_links/editor
> -    //= require "pageflow/internal_links/editor"

Sprockets does not use quotes in requires, does it?

**Comment by tilsammans** [comment]
Updated the PR per your comment.

**Comment by tf** [comment]
Thanks!

**Pull request merged**